src/encord_active/lib/common/utils.py

```diff
@@ -83,17 +84,12 @@ def load_json(json_file: Path) -> Optional[dict]:
             return None
 
 
-def get_du_size(data_unit: dict, img_pth: Optional[Path] = None) -> Optional[Tuple[int, int]]:
+def get_du_size(data_unit: dict, image: Optional[Image.Image] = None) -> Optional[Tuple[int, int]]:
     if "width" in data_unit and "height" in data_unit:
         return int(data_unit["height"]), int(data_unit["width"])
 
-    image_corrupted = False
-    if img_pth is not None and img_pth.is_file():
-        try:
-            image = cv2.imread(img_pth.as_posix())
-            return image.shape[:2]
-        except Exception:
-            image_corrupted = True
+    if image is not None:
+        return image.size
```

Comment:
Pillow's `Image.size` is a tuple `(widht, height)` which is the oposite order of what's returned elsewhere.